lib/ronin/exploits/cli/commands/run.rb

```diff
@@ -382,6 +391,24 @@ def run_exploit
             end
           end
 
+          #
+          # Run the exploit's test method, and print the result.
+          #
+          def run_test
+            case (result = @exploit.test)
+            when TestResult::Vulnerable
+              print_positive "Vulnerable: #{result}"
+            when TestResult::NotVulnerable
+              print_negative "NotVulnerable: #{result}"
+            when TestResult::Unknown
+              print_warning "Unknown: #{result}"
+            else
+              print_error "Unexpected result: #{result.inspect}"
+            end
+
+            result
```

Comment:
Do we need to return the `result` here?

Reply:
I figured it was better to return it in case the caller wanted to act on the result, rather than return the case statement's value.